server/api/oauthEnabledClient.js

```diff
@@ -1,7 +1,24 @@
 const axios = require('axios')
+const superagent = require('superagent')
 const { logger } = require('../services/logger')
 
-const { addAuthorizationHeader, addPaginationHeaders } = require('./axios-config-decorators')
+const { addAuthorizationHeader, addPaginationHeaders, getHeaders } = require('./axios-config-decorators')
+
+const superAgentResultHandler = result => {
```

Comment:
Not sure on the name handler for error and result, maybe getSuperAgentResult and getSuperAgentError?

Reply:
mmm... the `result` and `error` are passed into the functions, so they don't really do a `get`.  They are effectively side effect functions that just log and return the input